dbus-serialbattery/dbus-serialbattery.py

```diff
@@ -336,7 +336,7 @@ def check_bms_types(supported_bms_types, type) -> None:
                 battery[0] = testbms
 
     # CAN
-    elif port.startswith("can") or port.startswith("vecan"):
+    elif port.startswith(("can", "vecan", "vcan")):
```

Comment:
Is `vcan` used for external USB to CAN adapters or Raspberry Pi hats?

Reply:
No, vcan is the virtual can network that I use for the local simulation and tests. Can be also used to connect to remote can ports on other devices for example (socketcan ober network)